### DISCUSSION TITLE
Speading up the validTouch() function.

### DISCUSSION THREAD

**Discussion by FingolSE** [Bodmer/TFT_eSPI#1565] [Q&A - Touch]
When running my touch display with TFT_eSPI library the checking of the touch display is rather slow. It may be due too some induced signal triggering the while loop "Wait until pressure stops increasing to debounce pressure". By adding a requirement that z1 needs to be larger than z2+5 lowers the response time from validTouch by a factor of 3 in my HW environment.

wbr Håkan

**Comment by Bodmer** [comment]
Thanks for the tip. I am surprised that the change has such a significant effect.
